### DISCUSSION TITLE
Incidents Status Not Changing

### DISCUSSION THREAD

**Discussion by alexis-stallman** [AdhocAdam/smletsexchangeconnector#460] [Q&A] [answered]
Hello, we are having some issues where when someone comments on a resolved incident by sending an email, the comment is being added, but the incident's status is not changing to Active even though we have these settings picked:

<img width="391" alt="MicrosoftTeams-image (3)" src="https://user-images.githubusercontent.com/102493947/236303944-6ce534cf-ee73-47aa-abfb-c4411d9169dd.png">

Am I not understanding these settings correctly or is something not working?  Thanks!

**Comment by AdhocAdam** [comment, marked as answer]
Great question!

This feature only works as long as the Work Item is **not** Resolved or Closed. The rationale behind this being, if it were allowed it would open a way to bypass the formal reactivation of the Work Item wherein the following fields would be reset/nulled out:
- Status would be set to Active
- Resolution Date
- Resolution Category
- Resolution Description

**Reply by alexis-stallman** [reply]
That clears things up, thank you!
